azure-iot-device/tests/iothub/aio/test_async_clients.py

```diff
@@ -16,9 +16,14 @@
 from azure.iot.device.iothub.aio.async_inbox import AsyncClientInbox
 from azure.iot.device.common import async_adapter
 from azure.iot.device.iothub.auth import IoTEdgeError
+from azure.iot.device.common.models.x509 import X509
 
 # auth_provider and pipeline fixtures are implicitly included
-
+hostname = "beauxbatons.academy-net"
+device_id = "MyPensieve"
+fake_x509_cert_value = "fantastic_beasts"
```

Comment:
As in the sync tests, have a shared fixture defined in the client_fixtures to avoid duplication.